app/views/exhibits/fenway-park.md

```diff
@@ -0,0 +1,34 @@
+# Rear Bumper II: Jean Shepherd at Fenway Park
+
+1
```

Comment:
@caseyedavis12 : I'll go ahead and merge, but do you want all of these at the top of the page?